src/components/Form/Buttons/styles/CancelButton.style.tsx

```diff
@@ -0,0 +1,27 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+export const useStyles = makeStyles(
+  (theme: Theme) => ({
+    cancelButton: {
+      flex: '0 1 300px',
+      backgroundColor: theme.palette.info.light,
+      '& .MuiButton-label': {
+        color: theme.palette.common.white,
+        fontWeight: 700,
+        fontSize: '18px',
+        fontFamily: 'Raleway',
```

Comment:
do CancelButton and BasicButton use different font-family?

Reply:
No, we have the same Raleway font-family

Reply:
then is it needed here? or font-family is hard-coded  in every button variation instead of go in global styles for `.Mui-Button-label` ?

Reply:
We have hard-coded in every button, but i think we can make it global cause every variation uses the same font-family

Reply:
well, then it's up to you/team to decide whether it's worth to refactor right here or wait for the better moment
I don't have strong opinion on this